pyscf/adc/radc.py

```diff
@@ -602,6 +643,20 @@ def ea_adc(self, nroots=1, guess=None, eris=None):
     def ip_adc(self, nroots=1, guess=None, eris=None):
         return RADCIP(self).kernel(nroots, guess, eris)
 
+    def density_fit(self, auxbasis=None, with_df = None):
+        if with_df is None:
+            self.with_df = df.DF(self._scf.mol)
+            self.with_df.max_memory = self.max_memory
+            self.with_df.stdout = self.stdout
+            self.with_df.verbose = self.verbose
+            if auxbasis is None:
+                self.with_df.auxbasis = self._scf.with_df.auxbasis
+            else :
+                self.with_df.auxbasis = auxbasis
+        else :
+            self.with_df = with_df
```

Comment:
What is the purpose of with_df here? Unless with_df = None, all this function does is setting self.with_df = with_df. However, I may be missing the purpose of with_df.